frontend/public/components/limit-range.tsx

```diff
@@ -54,49 +55,51 @@ export const LimitRangeTableRow: RowFunction<K8sResourceKind> = ({ obj, index, k
   );
 };
 
-export const LimitRangeTableHeader = () => {
-  return [
-    {
-      title: 'Name',
-      sortField: 'metadata.name',
-      transforms: [sortable],
-      props: { className: tableColumnClasses[0] },
-    },
-    {
-      title: 'Namespace',
-      sortField: 'metadata.namespace',
-      transforms: [sortable],
-      props: { className: tableColumnClasses[1] },
-      id: 'namespace',
-    },
-    {
-      title: 'Created',
-      sortField: 'metadata.creationTimestamp',
-      transforms: [sortable],
-      props: { className: tableColumnClasses[2] },
-    },
-    {
-      title: '',
-      props: { className: tableColumnClasses[3] },
-    },
-  ];
-};
-LimitRangeTableHeader.displayName = 'LimitRangeTableHeader';
+export const LimitRangeList: React.FC = (props) => {
+  const { t } = useTranslation();
+  const LimitRangeTableHeader = () => {
+    return [
+      {
+        title: t('limit-range~Name'),
+        sortField: 'metadata.name',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[0] },
+      },
+      {
+        title: t('limit-range~Namespace'),
+        sortField: 'metadata.namespace',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[1] },
+        id: 'namespace',
+      },
+      {
+        title: t('limit-range~Created'),
+        sortField: 'metadata.creationTimestamp',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[2] },
+      },
+      {
+        title: '',
+        props: { className: tableColumnClasses[3] },
+      },
+    ];
+  };
 
-export const LimitRangeList: React.SFC = (props) => (
-  <Table
-    {...props}
-    aria-label="Limit Ranges"
-    Header={LimitRangeTableHeader}
-    Row={LimitRangeTableRow}
-    virtualize
-  />
-);
+  return (
+    <Table
+      {...props}
+      aria-label={LimitRangeModel.labelPlural}
```

Comment:
Aria-labels should be translated as well; I spoke to the accessibility folks in UXD about this and they said it was important. This also applies to aria-placeholder, aria-roledescription, or aria-valuetext.

Reply:
This label is the resource name, which should not be translated.

Reply:
Yup, I meant in general.